lib/helper/Playwright.js

```diff
@@ -282,6 +284,12 @@ class Playwright extends Helper {
     const contexts = await this.browser.contexts();
     contexts.shift();
 
+    // ensure current page is in default context
+    if (this.page) {
+      const existingPages = await this.browserContext.pages();
+      await this._setPage(existingPages[0]);
```

Comment:
Looks like this line causes Travis to fail 
page is set at the moment it is closed.
To be explicit - running `page.$('body')` throws error, again, because page is not defined anymore.

Please fix this.

Reply:
I believe the problem was since the Helper was set to restart the `_stopBrowser()` method was being called after I tried to reset the page to the default session, moving my `_setPage` to occur after that check seems to fix the issue.